packages/mui-styles/package.json

```diff
@@ -65,6 +56,29 @@
     "jss-plugin-vendor-prefixer": "^10.10.0",
     "prop-types": "^15.8.1"
   },
+  "devDependencies": {
+    "@mui/joy": "5.0.0-beta.5",
+    "@mui/material": "^5.14.8",
+    "@mui/styles": "^5.14.7",
+    "@types/chai": "^4.3.5",
+    "@types/react": "^18.2.21",
+    "@types/react-dom": "^18.2.7",
+    "@types/sinon": "^10.0.16",
+    "chai": "^4.3.7",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "sinon": "^15.2.0",
+    "test": "^5.0.0"
+  },
+  "peerDependencies": {
+    "@types/react": "^17.0.0 || ^18.0.0",
+    "react": "^17.0.0"
```

Comment:
Are you aware of incompatibilities of this package with react 18? I think it's the only peer dependency conflict we experience on Toolpad. We import for the docs. afaik our docs are on React 18, and this doesn't seem to be breaking.

Reply:
I don't see why Toolpad would have this dependency, removed: https://github.com/mui/mui-toolpad/pull/2684.

Reply:
Yep, I've tried the same before 😄

Reply:
> Are you aware of incompatibilities of this package with react 18?

@mnajdova would likely know the best.

Reply:
> Are you aware of incompatibilities of this package with react 18?

One issue is GlobalStyle injection that is called twice in React 18 so not removed after unmounting.